R/transform.R

```diff
@@ -96,6 +95,15 @@ visit <- function(pd_nested, funs) {
   pd_transformed
 }
 
+#' @rdname visit
+visit_post <- function(pd_nested, funs) {
```

Comment:
I have no idea why I put it here, but I meant to put it in close proximity to `visit()`.